pdf/pdfrenderthread.cpp

```diff
@@ -86,18 +87,25 @@ class SearchThread: public QThread
             }
 
             delete page;
+
+            if ((i + 1) % 3 == 0 || i + 1 == m_document->numPages()) {
+                emit searchProgress(float(i) / float(m_document->numPages() - 1),
```

Comment:
Divide by zero with documents of one page? 

Btw, currently getting toolbar on those requires zooming so it can be scrolled :)